src/cli/index.ts

```
@@ -0,0 +1,154 @@
+}
+
+async function runCommand(command: string, args: string[]) {
+	try {
+		switch (command) {
+			case 'config':
+				// Import and run the config module
+				await import('./config.js');
+				break;
+				
+			case 'chat':
+				const { ChatInterface } = await import('./chat.js');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)